src/exercise/02.html

```diff
@@ -5,7 +5,13 @@
 
   <script type="module">
     const rootElement = document.getElementById('root')
-    const reactElement = React.createElement('div', {children: 'Hello world!', className: 'container',})
+    const reactElement = React.createElement('div', {className: 'container',
+    	children:[
+    	React.createElement('span', null, 'Hello'),
```

Comment:
There's a funny thing that inspecting a console will show you, and that's that when we're including multiple child elements (like we are here) in an array, React wants them to all have unique `key` props... 

![Screen Shot 2021-06-21 at 2 49 00 PM](https://user-images.githubusercontent.com/7146063/122826320-38887780-d2a0-11eb-83bc-c2f2062f39f3.png)

There are so many completely valid ways to do this, and you've supplied one of them, but it's quite rare to  use `null` for those props.

In the `React.createElement` function, consider tossing out some 'keys' as props...  eg.

```
    const reactElement = React.createElement(
      'div',
      {
        className: 'container',
        children: [
          React.createElement(
            'span',
            {
              key: 'hello',
              children: 'Hello'
            }
          ),
          React.createElement(
            'span',
            {
              key: 'world',
              children: 'World'
            }
          )
        ]
      },
    );
```

This could also be written:
```
    const reactElement = React.createElement(
      'div',
      {
        className: 'container',
        children: [
          React.createElement(
            'span',
            {
              key: 'hello',
            },
            'Hello'
          ),
          React.createElement(
            'span',
            {
              key: 'world',
            },
            'World'
          )
        ]
      },
    );
```

... so many ways... so many, many ways.